miso-web/src/main/webapp/scripts/list_pool.js

```diff
@@ -62,15 +62,37 @@ ListTarget.pool = {
             required: true
           };
         });
+
+	      fields.push({
+          property: 'createBox',
```

Comment:
With how many times this field is repeated, it would be best to abstract it to somewhere as well - probably `ListUtils`